scipy/interpolate/_bsplines.py

```diff
@@ -561,6 +589,96 @@ def integrate(self, a, b, extrapolate=None):
         integral *= sign
         return integral.reshape(ca.shape[1:])
 
+    @classmethod
+    def from_power_basis(cls, pp, bc_type='not-a-knot'):
+        r"""
+        Construct a polynomial in the B-spline basis
+        from a piecewise polynomial in the power basis.
+        
+        For now, accepts ``CubicSpline`` instances only.
+
+        Parameters
+        ----------
+        pp : CubicSpline
+            A piecewise polynomial in the power basis, as created by CubicSpline
+        bc_type : string, optional
+            Boundary condition type as in CubicSpline:
+            one of the ``not-a-knot``, ``natural``, ``clamped``, or ``periodic``
+            ``bc_type`` is needed for composition a knot vector.
+
+        Returns
+        -------
+        b : BSpline object
+            A new instance representing the initial polynomial in the B-spline basis.
+
+        Notes
+        -----        
+        The algorithm follows from differentiation the Marsden’s identity [1]:   
+        each of coefficients of spline interpolation function in B-spline basis is 
+        computed as follows:
+        
+        .. math::
+        
+            c_j = \sum_{m=0}^{k} \frac{(k-m)!}{k!} c_{m,i} (-1)^{k-m} D^m p_{j,k}(x_i)
+        
+        :math: `c_{m, i}` - a coefficient of CubicSpline,
+        :math: `D^m p_{j, k}(x_i)` - an m-th defivative of a dual polynomial in ``x_i``.
+        ``k`` always equals 3 for now.
+        
+        First ``n - 2`` coefficients are computed in ``x_i = x_j``, e.g. 
+        
+        .. math::
+        
+            c_1 = \sum_{m=0}^{k} \frac{(k-1)!}{k!} c_{m,1} D^m p_{j,3}(x_1)
+        
+        Last ``nod + 2`` coefficients are computed ``in x[-2]``, 
+        ``nod`` - number of derivatives at the ends.
+         
+        Examples
+        --------
+        Consider x = [0, 1, 2, 3, 4], y = [1, 1, 1, 1, 1] and bc_type = ``natural``
+        The coefficients of Cubic Spline in the power basis:
+        
+        [[0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0],
+         [1, 1, 1, 1, 1]]
+	
+        The knot vector t = [0, 0, 0, 0, 1, 2, 3, 4, 4, 4, 4]
+        In this case c[j] = 0!/k!*c_{3, i}*k! = c_{3, i} = 1 for j = 0, ..., 6
```

Comment:
Examples section is usually for code examples. This snippet is more appropriate for the Notes section. Either way, if you keep this paragraph, please typeset code as code and formulas as TeX. (otherwise, it's going to look wrong in html and unshielded `*`-s and `_`-s will do unintended things).